hack/build.sh

```diff
@@ -81,7 +81,9 @@ run() {
 
 go_fmt() {
   echo "🧹 ${S}Format"
-  go fmt ./cmd/... ./pkg/...
+  gofmt -w $(find -name '*.go' | grep -v vendor)
```

Comment:
is 'gofmt' always available ? (go fmt is) If not we would need to check for it, too.

Also, I'd prefer a positive list (cmd, pgk, e2e) instead of a black list (-v vendor). To easily thing could slip through otherwise.

Reply:
I think its best to define a script variable like "source_dirs" which point to the top-level source directories and use that in every check/formatting command.

Reply:
```
source_dirs="cmd pkg test"
```

Reply:
`gofmt` should be always available as it's packaged along with other golang binaries shipped part of golang installation, also go fmt calls `gofmt -w -l` under the cover

Reply:
explicit `source_dir` sounds good, updated.